test_scripts/API/GetWayPoints/common_get_way_points.lua

```diff
@@ -0,0 +1,273 @@
+  common_get_way_points.rai_ptu_n(1, ptu_update_func, self)
+end
+
+function common_get_way_points.rai_ptu_n(id, ptu_update_func, self)
+  self, id, ptu_update_func = common_get_way_points.getSelfAndParams(id, ptu_update_func, self)
+  if not id then id = 1 end
+  self["mobileSession" .. id] = mobile_session.MobileSession(self, self.mobileConnection)
+  self["mobileSession" .. id]:StartService(7)
+  :Do(function()
+      local corId = self["mobileSession" .. id]:SendRPC("RegisterAppInterface", config["application" .. id].registerAppInterfaceParams)
+      EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config["application" .. id].registerAppInterfaceParams.appName } })
+      :Do(function(_, d1)
+          hmiAppIds[config["application" .. id].registerAppInterfaceParams.appID] = d1.params.application.appID
+          EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", { status = "UPDATE_NEEDED" }, { status = "UPDATING" }, { status = "UP_TO_DATE" })
+          :Times(3)
+          EXPECT_HMICALL("BasicCommunication.PolicyUpdate")
+          :Do(function(_, d2)
+              self.hmiConnection:SendResponse(d2.id, d2.method, "SUCCESS", { })
+              ptu_table = jsonFileToTable(d2.params.file)
+              ptu(self, ptu_update_func)
+            end)
+        end)
+      self["mobileSession" .. id]:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+      :Do(function()
+          self["mobileSession" .. id]:ExpectNotification("OnHMIStatus", { hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" })
+          :Times(1)
+          self["mobileSession" .. id]:ExpectNotification("OnPermissionsChange")
+        end)
+    end)
+end
+
+function common_get_way_points.rai_n(id, self)
+  self, id = common_get_way_points.getSelfAndParams(id, self)
+  if not id then id = 1 end
+  self["mobileSession" .. id] = mobile_session.MobileSession(self, self.mobileConnection)
+  self["mobileSession" .. id]:StartService(7)
+  :Do(function()
+      local corId = self["mobileSession" .. id]:SendRPC("RegisterAppInterface", config["application" .. id].registerAppInterfaceParams)
+      EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config["application" .. id].registerAppInterfaceParams.appName } })
+      :Do(function(_, d1)
+          hmiAppIds[config["application" .. id].registerAppInterfaceParams.appID] = d1.params.application.appID
+        end)
+      self["mobileSession" .. id]:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+      :Do(function()
+          self["mobileSession" .. id]:ExpectNotification("OnHMIStatus", { hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" })
+          :Times(1)
+          self["mobileSession" .. id]:ExpectNotification("OnPermissionsChange")
+        end)
+    end)
+end
+
+
+function common_get_way_points.start(pHMIParams, self)
```

Comment:
@LitvinenkoIra Please use the same logic as in commonRC.start()
This allows to run scripts on EXTERNAL_PROPRIETARY flow as well as on PROPRIETARY